internals-js/src/operations.ts

```diff
@@ -298,17 +337,42 @@ export class Field<TArgs extends {[key: string]: any} = {[key: string]: any}> ex
   }
 }
 
+/**
```

Comment:
That comment is technically wrapped, so I assume you imply that you'd prefer wrapping earlier/at shorter lines (I happen to not use VS Code, but my editor is still able to wrap lines wherever I ask it to)?

I didn't feel we had clear agreed upon code style formatting rules in this project so I'll admit I've done "my thing" so far (but I'm sorry if I missed something). That said, in principle, I'm happy to adapt to anything (and I'm not even against auto-formatters fwiw). Whatever gets us the fastest to a place where we don't have formatting discussions in reviews has my vote :D.

Reply:
Yeah, not sure there's any alignment on style (and if there was it should be enforced by tooling or why bother). 80 character wrap is common if you can and want to tweak your editor but feel free to disregard since we have no formalized convention.